
```{r, include = FALSE}
ottrpal::set_knitr_image_path()
```




# RNA Methods

```{r, fig.alt = "I need to put a description here", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g14391bffede_1_0")
```

## Learning Objectives

```{r, fig.alt = "Learning objectives This chapter will demonstrate how to: Understand the goals and data collection processes for gene expression assays. Compare and contrast the following methods: Bulk RNA-seq, Single cell RNA-seq, Gene expression microarrays", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g12890ae15d7_0_76")
```

## What are the goals of gene expression analysis?

The goal of gene expression analysis is to quantify RNAs across the genome. This can signify the extent to which various RNAs are being transcribed in a particular cell. This can be informative for what kinds of activity a cell is undergoing and responding to.

```{r, fig.alt = "The goal of gene expression analysis is to quantify RNAs on a genome wide level", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g142c259a793_0_0")
```

## Comparison of RNA methods

There are three general methods we will discuss for evaluating gene expression. RNA sequencing (whether bulk or single-cell) allows you to catch more targets than gene expression microarrays but is much more costly and computationally intensive. Gene expression microarrays have a lower dynamic range than RNA-seq generally but are much more cost effective.

```{r, fig.alt = "Gene expression microarrays are low cost and low computationally intensive. Bulk RNA-seq is higher cost, requires more computational resources but covers more targets than gene expression arrays. Single cell RNA-seq is higher cost, requires more computational resources but as opposed to Bulk RNA-seq gives single cell resolution.", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g13438a9a5b2_0_80")
```


## Gene expression microarrays


### Where gene expression array data comes from

```{r, fig.alt = "Microarray data is generation using total RNA which is labeled with fluorescent dyes. These are then hybridized to the microarray chip. The higher the fluorescence, the more RNA that is being detected in the sample. This is assessed chip wide by an image taken of the chip.", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g142e3de7ce8_0_8")
```

[Refine.bio](https://alexslemonade.github.io/refinebio-examples/02-microarray/00-intro-to-microarray.html) summarizes the microarray strengths and limitations.

>### Microarray data **strengths**:
- Microarrays historically were less expensive than RNA-seq allowing for more replicates and greater statistical power [@Tarca2006].
- Microarrays generally had a faster turn-around than RNA-seq [@LCSciences2014].

>As a result of these historical advantages, vast quantities of data have been generated worldwide using microarrays.
The microarray data compiled by refine.bio includes over 500,000 individual samples across over 25,000 experiments.
For many scientific questions, the best available gene expression data may be microarray based!

> ### Microarray data **limitations**:

> - If a transcript doesn't have a probe designed to it on a microarray, it won't be measured; standard microarrays can't be used for transcript discovery [@Mantione2014].
- A chip's probe designs are only as up to date as the genome annotation at the time it was designed [@Mantione2014].
- As is true for all techniques that involve nucleotide hybridization (RNA-seq too); microarray probes come with some biases depending on their nucleotide sequence composition (like GC bias).

### More about gene expression microarrays

- [Getting started in gene expression microarray analysis](https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1000543) [@Slonim2009].
- [Microarray and its applications](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3467903/) [@Govindarajan2012].
- [An tutorial review of microarray data analysis](http://www.ub.edu/stat/docencia/bioinformatica/microarrays/ADM/slides/A_Tutorial_Review_of_Microarray_data_Analysis_17-06-08.pdf) [@Sanchez2008].
- [Analysis of microarray experiments of gene expression profiling](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2435252/) [@Tarca2006].
- [Some nice summary slides about microarray data analysis from Emory](http://web1.sph.emory.edu/users/hwu30/teaching/bioc/GE1.pdf) [@intro-microarray-slides].

## Bulk RNA-seq


### Where RNA-seq data comes from


```{r, fig.alt = "Bulk RNA-seq data is generated by extracting total RNA and then isolating RNA specific species by either Poly-A selection, Ribo depletion, or size selection. The isolated RNA is then converted to cDNA so it is more stable for sequencing. This cDNA is used to construct a sequencing library. Lastly PCR amplification is used to make many copies to use for sequencing.", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g142c259a793_0_5")
```

The following material is quoted from [refine.bio](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html).

> ### RNA-seq data **strengths**  
- RNA-seq can assay unknown transcripts, as it is not bound to a pre-determined set of probes like microarrays [@Zhong2009].
- Its values are considered more dynamic than microarray values which are constrained to a smaller range based on background signal and probe sets being saturated [@Zhong2009].

> ### RNA-seq data **limitations/biases**  

> The nature of sequencing introduces several different kinds of biases:

> - **GC bias**: higher GC content sequences are less likely to be observed.  
- **3' bias (positional bias)**: for most sequencing methods, the 3 prime end of transcripts are more likely to be observed.  
- **Complexity bias**: some sequences are easier to be bound and amplified than others.   
- **Library size or sequencing depth**: the total number of reads is not always equivalent between samples.  
- **Gene length**: longer genes are more likely to be observed.   

> @bias-blog discusses these biases in this [blog post](https://mikelove.wordpress.com/2016/09/26/rna-seq-fragment-sequence-bias/) which includes this handy figure.

```{r, fig.alt = "RNA-seq data has various biases introduced to the data upon data generation. RNA targets are more likely to be picked up if they are long, if they are from the 3 prime end, have a particular GC content and have a particular read start sequence.", echo = FALSE}
ottrpal::include_slide("https://docs.google.com/presentation/d/1YwxXy2rnUgbx_7B7ENH9wpDX-j6JpJz6lGVzOkjo0qY/edit#slide=id.g142e3de7ce8_0_19")
```

### More about RNA-seq
- [Refine.bio's introduction to RNA-seq](https://alexslemonade.github.io/refinebio-examples/03-rnaseq/00-intro-to-rnaseq.html)
- [StatQuest: A gentle introduction to RNA-seq](https://www.youtube.com/watch?v=tlf6wYJrwKY) [@Starmer2017-rnaseq].
- [A general background on the wet lab methods of RNA-seq](https://bitesizebio.com/13542/what-everyone-should-know-about-rna-seq/) [@Hadfield2016].
- [Modeling of RNA-seq fragment sequence bias reduces systematic errors in transcript abundance estimation](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5143225/) [@Love2016].
- [Mike Love blog post about sequencing biases]( https://mikelove.wordpress.com/2016/09/26/rna-seq-fragment-sequence-bias/) [@bias-blog]
- [Biases in Illumina transcriptome sequencing caused by random hexamer priming](https://pdfs.semanticscholar.org/9d16/997f5de72d6c606fef3d673db70e5d1d8e1e.pdf?_ga=2.131436679.965169313.1600175795-124991789.1600175795) [@Hansen2010].
- [Computation for RNA-seq and ChIP-seq studies](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4121056/) [@Pepke2009].

## Single-cell RNA-seq

Single-cell technologies can be divided into two categories, tag-based and full-length, based on their capture methods and quantitative nature.

> *Tag-based* scRNA-seq, cells are separated by emulsion/droplets, and individual cells are given a unique cell barcode prior to sequencing.
An example of tag-based scRNA-seq is 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601).

> *Full-length* scRNA-seq, cells are physically separated into individual wells of a plate and are often also sorted by other means (e.g., Fluorescence Activated Cell Sorting).
With full-length scRNA-seq, each cell is sequenced individually and has its own fastq file.
An example of full-length scRNA-seq is Smart-seq2 [(Picelli _et al._ 2014)](https://www.nature.com/articles/nprot.2014.006).

> ### Tag-based scRNA-seq  
*Example:* 10X Genomics [(Zheng _et al._ 2017)](https://www.ncbi.nlm.nih.gov/pubmed/28091601)
Individual cells are separated by emulsion/droplets prior to cell lysis.
Transcripts from each cell are then tagged with two barcodes: a cell-specific barcode and a Unique Molecular Identifier (UMI) [(Islam _et al._ 2014)](http://www.nature.com/doifinder/10.1038/nmeth.2772).
All transcripts from all cells are then pooled together and undergo PCR amplification and sequencing as if they are one sample.

> Tagging of each transcript with a different UMI before amplification allows the identification of PCR duplicates, allowing control for PCR amplification errors and biases.
Individual samples have two fastq files: one for the cell and UMI barcodes (R1) and another with the transcript sequence reads (R2).

> #### Pros:  
- Can run potentially millions of cells at once.   
- Much less computationally demanding.
- Won't take up all your computer's storage.  
- Much cheaper.  

> #### Cons:  
- Sequencing is not bidirectional so data will likely have more intense 3' bias.  
- The sequencing depth per cell with these technologies is generally lower.

### More reading about single-cell RNA-seq data

- Hemberg Lab [scRNA-seq training course](https://scrnaseq-course.cog.sanger.ac.uk/website/index.html)
- [ASAP: Automated Single-cell Analysis Pipeline](https://asap.epfl.ch/) is a web server that allows you to process scRNA-seq data. ([Gardeux _et al._ 2017.](https://doi.org/10.1093/bioinformatics/btx337 ))
- Smith. [_Unique Molecular Identifiers â€“ the problem, the solution and the proof_](https://cgatoxford.wordpress.com/2015/08/14/unique-molecular-identifiers-the-problem-the-solution-and-the-proof/) provides an excellent background on UMIs.
- [Baran-Gale _et al._ (2018)](https://doi.org/10.1093/bfgp/elx035)  
- [Lafzi _et al._ (2019)](https://doi.org/10.1038/s41596-018-0073-y)
- [Ziegenhain _et al._ (2018)](http://dx.doi.org/10.1016/j.molcel.2017.01.023)
- [Zhang _et al._ (2018)](https://doi.org/10.1016/j.molcel.2018.10.020)
- [AlJanahi _et al._ (2018)](https://doi.org/10.1016/j.omtm.2018.07.003)
- [Amezquita _et al._ (2019)](https://www.nature.com/articles/s41592-019-0654-x)
- [Angerer _et al._ (2017)](http://dx.doi.org/10.1016/j.coisb.2017.07.004)
- [Bruning _et al._ (2021)](https://www.biorxiv.org/content/10.1101/2021.02.15.430948v2)
- [Luecken & Theis (2019)](https://doi.org/10.15252/msb.20188746)
