
```{r, include = FALSE}
ottrpal::set_knitr_image_path()
```

# Amplicon Microbiome Sequencing

<div class = "warning">
This chapter is incomplete! If you wish to contribute, please [go to this form](https://forms.gle/dqYgmKH8XXE2ohwD9) or our [GitHub page](https://github.com/fhdsl/Choosing_Genomics_Tools).
</div>

## Learning Objectives

```{r, fig.alt = "FIGURE CAPTION GO HERE", out.width = "100%", echo = FALSE}
ottrpal::include_slide("GOOGLESLIDE GOES HERE")
```
## Goals of Amplicon analysis

## Data Collection and Processing of Amplicon Analysis
- QIIME 2 is a bioinformatics microbiome analysis platform. QIIME 2 has all the 
tools discussed in the course in one ENV. 
    - There are currently 3 different interfaces: Python, CLI and Galaxy! 
    - add cool features of QIIME 2 

## Upstream Analysis

### Demultiplexing

### Quality Filtering
- Dada2 
- Deblur 
- Cutadapt 


## Downstream Analysis 

### Phylogenetic Tree Construction

### Rarefaction
- Even sampling 
- Relative frequency (?)

### Alpha Diversity

### Beta Diversity

### Taxonomic Annotation
### Differential Abundance  
### Speciality Tools
#### Longitudinal Data 
#### Fecal Microbiota Transplant 
#### Predictive Models






## More resources

- [DNA methylation analysis with Galaxy tutorial](https://training.galaxyproject.org/training-material/topics/epigenetics/tutorials/methylation-seq/tutorial.html)
- [The mint pipeline](https://github.com/sartorlab/mint/blob/master/README.md) for analyzing methylation and hydroxymethylation data.
- [Book chapter about finding methylation regions of interest](https://compgenomr.github.io/book/extracting-interesting-regions-differential-methylation-and-segmentation.html)
